website/project/model.py

```diff
@@ -3308,6 +3297,15 @@ def get_admin_contributors_recursive(self, unique_users=False, *args, **kwargs):
                     else:
                         yield (contrib, node)
 
+    def get_admin_contributors(self, users):
+        """Return a set of all admin contributors for this node. Excludes contributors on node links and
```

Comment:
Minor: The bit about excluding contributors on node links isn't relevant for this method.